platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -37,9 +43,15 @@ OR
 
 ## What to expect from accessibility specialists during staging review
 
-After you request a staging review, the accessibility specialist reviews your work and documents any defects in ZenHub tickets. Issues with a 508 defect rating of “0” or “1” are launch blockers. Others can be fixed post-launch.  
+After you request a staging review, the accessibility specialist reviews your work and documents any defects in ZenHub tickets. **Issues with a 508 defect rating of “0” or “1” are launch blockers.** Issues with severity raings of "2, 3, or 4" should be fixed according to the following schedule:
```

Comment:
Consider moving the `**Issues with a 508 defect rating of “0” or “1” are launch blockers.** ` to be the first bullet, and then follow with the others. I'm worried it will get missed inside the paragraph. 

Also, consider bolding the when they should be fixed.